<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_br_open">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}
</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
    <property name="shortcut">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_br_close">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}
</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
    <property name="shortcut">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font: bold;
  color: white;
  font-size: 18px; 
  background-color: rgb(242, 163, 56);
  border: 1px solid black;
}

QPushButton:pressed {
     background-color: rgb(202, 126, 4 );
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="shortcut">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font: bold;
  color: white;
  font-size: 18px; 
  background-color: rgb(242, 163, 56);
  border: 1px solid black;
}

QPushButton:pressed {
     background-color: rgb(202, 126, 4 );
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="shortcut">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font: bold;
  color: white;
  font-size: 18px; 
  background-color: rgb(242, 163, 56);
  border: 1px solid black;
}

QPushButton:pressed {
     background-color: rgb(202, 126, 4 );
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="shortcut">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font: bold;
  color: white;
  font-size: 18px; 
  background-color: rgb(242, 163, 56);
  border: 1px solid black;
}

QPushButton:pressed {
     background-color: rgb(202, 126, 4 );
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="shortcut">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal_sign">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font: bold;
  color: white;
  font-size: 18px; 
  background-color: rgb(242, 163, 56);
  border: 1px solid black;
}

QPushButton:pressed {
     background-color: rgb(202, 126, 4 );
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>549</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>33</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  color: white;
  background-color : rgb(41, 41, 44);
  qproperty-aligment: 'AlignVCenter | AlignRight';
}

</string>
    </property>
    <property name="text">
     <string>0 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   color: white;
   font-size: 18px;
   background-color: rgb(103, 104, 104 );
   border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font: bold;
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="shortcut">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>xʸ</string>
    </property>
    <property name="shortcut">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>183</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>122</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>244</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}
</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>61</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>61</y>
      <width>183</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_show">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>122</y>
      <width>183</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
  color: white;
  background-color : rgb(41, 41, 44);
  qproperty-aligment: 'AlignVCenter | AlignRight';
  border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> enter x value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_credit">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>244</y>
      <width>183</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}
</string>
    </property>
    <property name="text">
     <string>credit</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deposit">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>305</y>
      <width>183</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font-size: 18px;
  color: rgb(227,227, 227);
  background-color: rgb(72, 73, 73);
  border: 1px solid black;
}
QPushButton:pressed {
     background-color: rgb(103, 104, 104 );
}
</string>
    </property>
    <property name="text">
     <string>deposit</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_max">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>204</y>
      <width>183</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
  color: white;
  background-color : rgb(41, 41, 44);
  qproperty-aligment: 'AlignVCenter | AlignRight';
  border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> enter x max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_min">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>163</y>
      <width>183</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
  color: white;
  background-color : rgb(41, 41, 44);
  qproperty-aligment: 'AlignVCenter | AlignRight';
  border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> enter x min</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
